Outlines/ens-beginners-guide.md

```diff
@@ -0,0 +1,27 @@
+# ENS: Beginner's Guide
+
+- ENS is an incredibly useful tool for dapp developers. 
+- The Ethereum Naming Service (ENS) is a decentralised naming service built on Ethereum.
+- It can be thought of like DNS.
+- It enables friendly names to be resolved to multiple targets.
+- A domain name has a root, and many subdomains separated by '.'
+- Each name can point to a single target or multiple targets depending on what you want to achieve. 
+- It is highly extensible and supports many target types such as Ethereum addresses/contracts, ABI definitions, content hashes (IPFS), public keys, and key/value text items
+- It lives on the Ethereum blockchain so is naturally decentralised.
+- It can be queried from smart contracts
+- By being able to resolve friendly names it enhances your application's usability.
+- Particularly good for reliably resolving contract addresses and their associated ABIs
+
+## Code Walkthrough
+
+- Many Ethereum libraries, such as Web3 or Ethers.js now support ENS out-of-the-box.
```

Comment:
@darrenlangley For now let's stick with Web3.js.

Reply:
All good.